.gitignore

```diff
@@ -84,3 +84,7 @@ tags
 # 3rd party downloads
 depends/steam
 depends/SDL2
+
+# lua typing
+.luarc.json
+/types
```

Comment:
is `types/` the standard location for these types of files? Who reads the output files and is there any setup required?

Reply:
LSP wiki do not specify exact naming or location. Only internal structure (`/library`, `config.json`) is strict. Location can be passed as arg (even otside the workspace).
`/definitions` can be an alternative to `/types`.

Reply:
As an addition, type hinting do not work in lua file, where `_ENV` is used. Dunno how to fix it yet.